### PR TITLE
Updated the elasticsearch part.

### PR DESCRIPTION
**Pull request by arpit1303**
Updated the terraform options to provision Elasticsearch cluster with dedicated master nodes and security plugin features.

### CODE REVIEW

File: examples/main.tf [hunk 1]
```diff
@@ -82,3 +82,54 @@ resource "instaclustr_vpc_peering" "example_vpc_peering" {
   peer_account_id = "1234567890"
   peer_subnet = "10.0.0.0/20"
 }
+
+resource "instaclustr_cluster" "example-elasticsearch" {
```
Comment (arjunrajshekhar) [inline examples/main.tf:86]: Could the 2 new examples here be reduced to 1?

File: README.md [hunk 2]
```diff
@@ -144,7 +144,10 @@ options|Options and add-ons for the given bundle. See `bundle.options` below for
 Property | Description | For Bundles | Default
 ---------|-------------|-------------|--------
 auth_n_authz|Accepts true/false. Enables Password Authentication and User Authorization.|Cassandra|false
-client_encryption|Accepts true/false. Enables Client ⇄ Node Encryption.|Cassandra, Kafka, Spark|false
+client_encryption|Accepts true/false. Enables Client ⇄ Node Encryption.|Cassandra, Kafka, Elasticsearch, Spark|false
+dedicated_master_nodes|Accepts true/false. Enables Dedicated Master Nodes.|Elasticsearch|false
+master_node_size|Desired master node sizes.|Elasticsearch|Required
+security_plugin|Accepts true/false. Enables Securiy Plugin.|Elasticsearch|false
```
Comment (arjunrajshekhar) [inline README.md:150]: Typo in "Securiy". Also could this parameter be explained with a bit more detail?

File: README.md [hunk 3]
```diff
@@ -144,7 +144,10 @@ options|Options and add-ons for the given bundle. See `bundle.options` below for
 Property | Description | For Bundles | Default
 ---------|-------------|-------------|--------
 auth_n_authz|Accepts true/false. Enables Password Authentication and User Authorization.|Cassandra|false
-client_encryption|Accepts true/false. Enables Client ⇄ Node Encryption.|Cassandra, Kafka, Spark|false
+client_encryption|Accepts true/false. Enables Client ⇄ Node Encryption.|Cassandra, Kafka, Elasticsearch, Spark|false
+dedicated_master_nodes|Accepts true/false. Enables Dedicated Master Nodes.|Elasticsearch|false
+master_node_size|Desired master node sizes.|Elasticsearch|Required
```
Comment (arjunrajshekhar) [inline README.md:149]: Could you add a reference to the options for node sizes? Please refer to how "node_sizes" is defined in the README

File: examples/main.tf (hunk 1 above)
Comment (arpit1303) [reply examples/main.tf:86]: Yep done.

File: README.md (hunk 2 above)
Comment (arpit1303) [reply README.md]: done

File: examples/main.tf (hunk 1 above)
Comment (arpit1303) [reply examples/main.tf:86]: done

File: README.md (hunk 3 above)
Comment (arpit1303) [reply README.md]: done

**Comment by arjunrajshekhar** [comment]
I have run the acceptance tests and they have passed.

**Pull request merged**